c_binding/lsm_datatypes.cpp

```diff
@@ -874,6 +876,28 @@ int lsm_system_mode_get(lsm_system *s, lsm_system_mode_type *mode)
         return LSM_ERR_NO_SUPPORT;
 }
 
+int lsm_system_read_cache_pct_set(lsm_system *sys, int read_pct)
+{
+    if ((sys == NULL) || (! LSM_IS_SYSTEM(sys)) ||
+        (read_pct == LSM_SYSTEM_CACHE_PCT_NO_SUPPORT))
+        return LSM_ERR_INVALID_ARGUMENT;
+
+    sys->read_cache_pct = read_pct;
+
+    return LSM_ERR_OK;
+}
+
+int lsm_system_read_cache_pct_get(lsm_system *s, int *read_pct)
+{
+    if ((s == NULL) || (read_pct == NULL) || (! LSM_IS_SYSTEM(s)))
+        return LSM_ERR_INVALID_ARGUMENT;
+
+    if (s->read_cache_pct != LSM_SYSTEM_CACHE_PCT_NO_SUPPORT)
+        *read_pct = s->read_cache_pct;
+
```

Comment:
When `s->read_cache_pct ==  LSM_SYSTEM_CACHE_PCT_NO_SUPPORT`  I was expecting that we would be returning `LSM_ERR_NO_SUPPORT` like we do for python?